Bottom: ecf099f1ef824d2dac36b1cac5a531f1f2cf8d7b
Top:    8672db3e5475f5ca2c35ec62a06866ddcf55eecd
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2021-01-09 16:26:58 +0200

update 5.10.x LIC_FILES_CHKSUM

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb b/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb
index 109ef3bcae..556cfe3bb5 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb
+++ b/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb
@@ -31,8 +31,8 @@
 inherit kernel
 require recipes-kernel/linux/linux-yocto.inc
 
-# some 4.19.x LIC_FILE_CHECKSUM: 
-LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+# some 5.10.x LIC_FILE_CHECKSUM:
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
 {{ if kernel_choice == "custom" and custom_kernel_remote == "y": }}
 SRC_URI = "{{=custom_kernel_remote_path}};protocol=git;bareclone=1;branch=${KBRANCH}"
